src/export/option.ts

```diff
@@ -108,9 +108,12 @@ interface ToolboxComponentOption extends ToolboxOption {
     }
 }
 
-export type DataZoomComponentOption = SliderDataZoomOption | InsideDataZoomOption;
-export type VisualMapComponentOption = ContinousVisualMapOption | PiecewiseVisualMapOption;
-export type LegendComponentOption = LegendOption | ScrollableLegendOption;
+export { SliderDataZoomComponentOption, InsideDataZoomComponentOption };
+export type DataZoomComponentOption = SliderDataZoomComponentOption | InsideDataZoomComponentOption;
+export { ContinousVisualMapComponentOption, PiecewiseVisualMapComponentOption };
+export type VisualMapComponentOption = ContinousVisualMapComponentOption | PiecewiseVisualMapComponentOption;
+export { PlainLegendOption, ScrollableLegendOption };
```

Comment:
Should be `PlainLegendComponentOption` and `ScrollableLegendComponentOption `

Reply:
Oh, no 🙈 
Sorry, I missed this one - now every touched type is suffixed with `ComponentOption`